components/ui-api-layer/docs/usage-kind-resources.md

```diff
@@ -0,0 +1,12 @@
+# Query usageKindResources
+
+he query returns all instances of the resources specified by the **usageKind** parameter in the given **environment**.
+The **metadata.ownerReference** field filters the result. It contains the resources which do not contain **metadata.ownerReference**.
```

Comment:
`The result contains the resources which do not have the **metadata.ownerReference**.`